src/libLookup/Lookup.cpp

```diff
@@ -4759,7 +4822,9 @@ bool Lookup::ToBlockMessage(unsigned char ins_byte) {
           ins_byte != LookupInstructionType::SETSTATEDELTAFROMSEED &&
           ins_byte != LookupInstructionType::SETSTATEDELTASFROMSEED &&
           ins_byte != LookupInstructionType::SETDIRBLOCKSFROMSEED &&
-          ins_byte != LookupInstructionType::SETMINERINFOFROMSEED);
+          ins_byte != LookupInstructionType::SETMINERINFOFROMSEED &&
+          ins_byte != LookupInstructionType::FORWARDTXN);  // Allow fwded txn to
```

Comment:
Here ,Adding forward txn here, do we mean to ignore such requests if the node is not synced?
I didn't get what is meant by comment `Allow fwded txn to to buffer`

Reply:
To allow/**Not-Block** `FORWARDTXN` msgs even if not synced.